submarine-runtime/tony-runtime/src/main/java/org/apache/submarine/runtimes/tony/TonyUtils.java

```diff
@@ -106,6 +106,7 @@ public static Configuration tonyConfFromClientContext(
     tonyConf.setStrings(TonyConfigurationKeys.CONTAINER_LAUNCH_ENV,
         envs.stream().map(env -> env.replaceAll("DOCKER_", ""))
             .toArray(String[]::new));
+    tonyConf.setStrings(TonyConfigurationKeys.APPLICATION_TYPE, "SUBMARINE");
```

Comment:
We'd better use the constant of `SUBMARINE`.

Reply:
thanks for reviewing,I will fix it